client/main/options.js

```diff
@@ -2,166 +2,178 @@
 
 'use strict';
 
-const Options = function() {
+const SuperClass = require('../common/superclass');
+const _ = require('underscore');
 
-};
+const OptionType = {
```

Comment:
what are your thoughts on the `Type` on the end of `OptionType`? By convention, anything which begins with a capital, i.e. `Option` is a type, so wouldn't `Option`, `Group`, `Terms`, etc. suffice?